neo/Network/P2P/Payloads/BulkInvPayload.cs

```diff
@@ -0,0 +1,120 @@
+using Neo.IO;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+
+namespace Neo.Network.P2P.Payloads
+{
+    public class BulkInvPayload : ISerializable
+    {
+        public const int MaxEntriesCount = 500;
+        public const int MaxSize = 512 * 1024; // 512Kb packet
```

Comment:
Hi! I think 512kb packet a little unreasonable. We should use something between 1400 bytes and 64kb.
From my experience, we should use 1400 bytes if we want to assure we are using only one TCP packet. 
MTU (ethernet) of 1500 bytes minus TCP headers = ~1400 bytes.